docs/SUBMISSION.md

```diff
@@ -21,7 +21,13 @@ Please refer to the inline comments in the base `application.yaml` configuration
 
 ## TAN Verification
 
-Coming soon
+When submitting diagnosis keys, a Transaction Authorization Number (TAN) token must be present in the request header section (`cwa-authorization`).
+Before delegating the TAN validation to the verification server, the TAN is verified to be an UUID on the Submission Service side.
+Then the TAN token is sent to the [verification server](https://github.com/corona-warn-app/cwa-verification-server/blob/master/docs/architecture-overview.md)
+to check its validity. If the TAN is valid, then it means it is linked to a valid test.
+In case the TAN is not valid, then the verification server will respond with `404 Not Found` and the Submission Service will respond with `HTTP 403 Forbidden`.
```

Comment:
```suggestion
In case the TAN is not valid, then the verification server will respond with `HTTP 404 Not Found` and the Submission Service will respond with `HTTP 403 Forbidden`.
```